src/protocols/secure_channel/tests/TestCheckinMsg.cpp

```diff
@@ -203,16 +283,16 @@ void TestCheckInMsg::TestCheckInMessageGenerate_EmptyAesKeyHandle(nlTestSuite *
     ByteSpan applicationData(vector.application_data, vector.application_data_len);
 
     /*
-        Passing an empty key handle while using PSA crypto will result in a failure.
-        When using OpenSSL this same test result in a success.
-        Issue #28986
-
-    Verify that the generation fails with an empty key handle
+        TODO(#28986): Passing an empty key handle while using PSA crypto will result in a failure.
```

Comment:
Which one is the desired outcome?

Reply:
Most the likely the one where the encryption function returns an error but the answer is not definitive. 

As 0 key is a valid key but and undefined psa_key_id is not... The issue tracks the investigation and resolution but it is not clear cut.